apps/nextra/pages/en/build/cli/use-cli.mdx

```diff
@@ -1,10 +0,0 @@
-# Use the Aptos CLI
-
-The Aptos command line interface (CLI) is a tool to help you compile and test Move contracts and quickly play with Aptos features on-chain.
-
-You can explore each of the use-cases here:
-
-1. [Working with Move Contracts](working-with-move-contracts.mdx)
-2. [Trying things on-chain](trying-things-on-chain.mdx)
```

Comment:
This file isn't used anymore?

Reply:
No, not since we flattened it. It used to be the page for the directory `use-cli` and had a bunch of subpages

Reply:
See https://aptos.dev/en/build/cli/use-cli